### PR TITLE
Feat/initialization design

### PR DESCRIPTION
**Pull request by kayiganwa04**
In this PR I implemented the ui design we have in figma both desktop and mobile ui
Note: now the form is disabled and some of the inputs
image
image

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  description: "Generated by create next app",
+}
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  return (
+    <html lang="en">
```
Comment (abelbordonado) [inline src/app/layout.tsx:17]: Not sure the lang need to be hardcoded here

File: src/app/page.tsx [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+                              <span className="text-black md:ml-4 ml-2 md:text-base text-xs text-center">Preview</span>
+                            </div>
+                            <span className="text-black md:text-base text-xs">5.7 MB</span>
+                          </div>
+                        </div>
+                        <div className="relative w-full mt-6 border border-gray-300 p-[0.7rem] px-6 rounded-lg">
+                          <label className="block mb-2 text-sm font-medium text-gray-900 absolute z-10 top-[-0.6rem] bg-white left-4 px-2">Identity / Passport Picture (Back)</label>
+                          <div className="flex mt-3">
+                            <div>
+                              <svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48" fill="none">
```
Comment (abelbordonado) [inline src/app/page.tsx:108]: Move the SVG to separated files to be imported

File: src/app/page.tsx [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+          <div className="relative bg-white md:w-[23%] w-full h-full ">
+            <div className="flex flex-col justify-center items-center my-4">
+              <Image
+                width={60}
+                height={48}
+                src={Logo}
+                className="w-60"
+                alt="logo"
+              />
+              <span className="text-black">Language: </span>
```
Comment (abelbordonado) [inline src/app/page.tsx:19]: Here are the flags comming right?

File: src/app/page.tsx [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+                                </svg>
+                              </div>
+                              <span className="text-black text-center md:ml-4 ml-2 md:text-base text-xs">Passport Front.png</span>
+                              <span className="text-black md:ml-4 ml-2 md:text-base text-xs text-center">Preview</span>
+                            </div>
+                            <span className="text-black md:text-base text-xs">5.7 MB</span>
+                          </div>
+                        </div>
+                        <div className="relative w-full mt-6 border border-gray-300 p-[0.7rem] px-6 rounded-lg">
+                          <label className="block mb-2 text-sm font-medium text-gray-900 absolute z-10 top-[-0.6rem] bg-white left-4 px-2">Identity / Passport Picture (Back)</label>
```
Comment (abelbordonado) [inline src/app/page.tsx:105]: The file input can be in `commons/inputs/fileInput`  as a generic component

File: src/app/page.tsx [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+                      <form className="" action="#">
+                        <div className="flex justify-between w-full">
+                          <div className="relative w-[48%]">
+                            <label className="block mb-2 text-sm font-medium text-gray-900 absolute z-10 bottom-[1.8rem] bg-white left-4 px-2">First name</label>
+                            <input type="email" name="email" id="email" className=" w-full border border-gray-300 text-black text-start p-[0.7rem] rounded-lg" placeholder="Your first name" />
+                          </div>
+                          <div className="relative w-[48%]">
+                            <label className="block mb-2 text-sm font-medium text-gray-900 absolute z-10 bottom-[1.8rem] bg-white left-4 px-2">Last name</label>
+                            <input type="email" name="email" id="email" className="w-full border border-gray-300 text-black text-start p-[0.7rem] rounded-lg" placeholder="Your last name" />
+                          </div>
```
Comment (abelbordonado) [inline src/app/page.tsx:68]: Lets make out of the inputs a generic component `common/inputs`

File: src/app/page.tsx [hunk 6]
```diff
@@ -0,0 +1,339 @@
...
+                        <div className="flex justify-between w-full mt-6">
+                          <div className="relative w-[48%]">
+                            <label className="block absolute px-2 text-black bottom-[2.7rem] z-10 bg-white left-4 text-sm font-medium">Adults passengers</label>
+                            <select id="countries" className="border border-gray-300 text-gray-900 text-sm rounded-lg block w-full p-4 ">
+                              <option selected>1</option>
+                              <option value="US">2</option>
+                              <option value="CA">3</option>
+                              <option value="FR">4</option>
+                              <option value="DE">5</option>
+                            </select>
```
Comment (abelbordonado) [inline src/app/page.tsx:147]: Selects are another common component

File: src/app/page.tsx [hunk 7]
```diff
@@ -0,0 +1,339 @@
...
+                          </div>
+                          <div className="relative w-[48%]">
+                            <label className="block px-2 absolute text-black bottom-[2.7rem] z-10 bg-white left-4 text-sm font-medium">Kids passengers</label>
```
Comment (abelbordonado) [inline src/app/page.tsx:150]: I feel we are abusing of tailwind CSS, we can create own CSS for really global things like the font-types

File: src/app/page.tsx [hunk 8]
```diff
@@ -0,0 +1,339 @@
...
+                          </div>
+                          <span className="text-sm ml-2">We require to know this information to prepare the lifevest
+                            and other equipment accordingly</span>
+                        </div>
+                      </form>
+                    </div>
+                  </div>
+                </div>
+
+                {/* Second form */}
```
Comment (abelbordonado) [inline src/app/page.tsx:175]: Each form can be in his own high-level components.
In this case will be interesting to save the data in a reducer 
(And both components will feel much more simple)

pages/booking/index
pages/booking/partial/booking-form-1
pages/booking/partial/booking-form-2

**Review by abelbordonado — COMMENTED** [review]
Visually looks amazing in the screen shot
Deploying netlify now

- Move the SVG to some common/svg folder
- Move the inputs (and make them generic components) in another folder. (We have to make other forms too)
- Create a booking page and 2 partials for each sub-form. You can create another partial for the lateral view

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

File: README.md [hunk 9]
```diff
@@ -1 +1,55 @@
-# mediterranea-marine-nextjs
\ No newline at end of file
+# Frontend Readme
```
Comment (abelbordonado) [inline README.md:1]: Dont just copy paste!! You need to Write it!!


🔥  💀

**Comment by abelbordonado** [comment]
I also miss the thanks page (which is another partial in the same page)

File: src/app/page.tsx (hunk 7 above)
Comment (kayiganwa04) [reply src/app/page.tsx:150]: sure

File: src/app/page.tsx (hunk 3 above)
Comment (kayiganwa04) [reply src/app/page.tsx:19]: yes

**Pull request merged**